src/app.js

```diff
@@ -66,6 +66,7 @@ socketController.setIo(io);
  * API Routes
  */
 app.post('/api/login', apiController.postLogin);
+app.post('/api/signup', apiController.postSignup);
```

Comment:
Do you mind changing these routes to just `login` and `signup`? Imo controllers should be api-independent.